src/app/modules/shared/components/month-picker/month-picker.component.spec.ts

```diff
@@ -69,4 +68,14 @@ describe('MonthPickerComponent', () => {
     expect(component.selectDate).toHaveBeenCalledWith(monthSelect, yearSelect);
   });
 
+  it('monthEnable set disable true in the buttoms futures', () => {
+    const monthFuture =  component.monthCurrent + 1;
+    expect(component.monthEnable(monthFuture)).toBeTrue();
+  });
+
+  it('monthEnable set disable false in the buttoms presents and past', () => {
```

Comment:
this description is not very clear:

I Understand this.
monthEnable set disabled to false in the months presents and past

I Think is no necessary disable the past months only future months